backend/src/api/aws_integration.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass
+import boto3
+import tempfile
+import os
+import uuid
+
+SOUND_BUCKET_NAME = os.environ["S3_SOUND_BUCKET"]
+IMAGE_BUCKET_NAME = os.environ["S3_IMAGE_BUCKET"]
+S3_SOUND_LOCATION = f"http://{SOUND_BUCKET_NAME}.s3.amazonaws.com/"
+S3_IMAGE_LOCATION = f"http://{IMAGE_BUCKET_NAME}.s3.amazonaws.com/"
+ALLOWED_SOUND_EXTENSIONS = {"mp3", "mp4", "m4a", "wav", "ogg"}
+ALLOWED_IMAGE_EXTENSIONS = {"pdf", "png", "jpg", "jpeg", "gif"}
+
+s3_session = boto3.Session(aws_access_key_id=os.environ["S3_KEY"], aws_secret_access_key=os.environ["S3_SECRET"])
+
+s3_client = s3_session.client("s3")  # pyright: ignore
+
+
+@dataclass(slots=True)
+class AWS_File:
+    filename: str
+    content_type: str
+    content: bytes
+
+    @property
+    def bucket(self) -> str:
+        if self.content_type == "audio/mpeg":
```

Comment:
this mimetype check is not going to work, as the mimetype of mp3 is different to opus/aac/flac etc but they are all sound files